lib/pos_blockchain.go

```diff
@@ -1133,12 +1133,12 @@ func (bav *UtxoView) hasValidProposerPartialSignaturePoS(block *MsgDeSoBlock, sn
 // block height + view number pair. It returns a bool indicating whether
 // we confirmed that the leader is valid. If we receive an error, we are unsure
 // if the leader is invalid or not, so we return false.
-func (bc *Blockchain) hasValidBlockProposerPoS(block *MsgDeSoBlock, parentUtxoView *UtxoView) (_isValidBlockProposer bool, _err error) {
-	currentEpochEntry, err := parentUtxoView.GetCurrentEpochEntry()
+func (utxoView *UtxoView) hasValidBlockProposerPoS(block *MsgDeSoBlock) (_isValidBlockProposer bool, _err error) {
```

Comment:
We generally name the receivers `bav`. Goland gets mad at me when the receiver names don't match.